exercises/practice/proverb/.meta/Proverb.example.ps1

```diff
@@ -0,0 +1,38 @@
+Function Get-Proverb() {
+    <#
+    .SYNOPSIS
+    Given a list of inputs, generate the relevant proverb.
+
+    .DESCRIPTION
+    Take a list of inputs and output the full text of a proverbial rhyme base on those inputs.
+
+    .PARAMETER Data
+    The list of inputs to generate the proverb.
+
+    .EXAMPLE
+    Get-Proverb -Input @("nail", "shoe", "horse", "rider", "message", "battle", "kingdom")
+
+    ```
```

Comment:
For multiline strings use:
```pwsh
@” 
line
another line
"@
```

Reply:
Ok. I fixed those.
Since you mention here-strings, I'm wonder if you know any trick or way to work with it in script file? 
I suppose it is fine inside the comment/example section, however I tried to use it with actual code and I ran into error because they are not allowed anything before the final "@ , which is some what of a small problem consider we often use space or tab to indent code.